packages/core/src/dowdep.ts

```diff
@@ -0,0 +1,43 @@
+import pathExists from 'path-exists'
+import path from 'path'
+
+import { NpmDependencySearcher } from './npm-dependencies'
+import { OnlyData } from './utils/OnlyData'
+import { PackageReferenceSearcher, ReferenceSearcherStrategy } from './references'
+import { Package } from './packages'
+import { Dependency, DependencySearcher } from './dependencies'
+
+
+export class Dowdep {
+    constructor(init: Partial<OnlyData<Dowdep>>) {
+        Object.assign(this, init)
+    }
+
+    dependencyLimit?: number
+    fileSystem: FileSystem = defaultFileSystem
+    githubAccessToken?: string
+    githubClient: any
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/LinqLover/downstream-repository-mining/pullRequest?prid=8058375)
```suggestion
    githubClient: unknown
```